hist/hist/src/TH1.cxx

```diff
@@ -2505,12 +2505,17 @@ void TH1::ClearUnderflowAndOverflow()
 ///  The resulting integral is normalized to 1.
 ///  If the routine is called with the onlyPositive flag set an error will
 ///  be produced in case of negative bin content and a NaN value returned
+///  \param onlyPositive If set to true, an error will be produced and NaN will be returned
+///  when a bin with negative number of entries is encountered.
+///  \param option
+///  - `""` (default) Compute the cumulative density function assuming current bin contents represent counts.
+///  - `"width"` Computes the cumulative density function assuming current bin contents represent densities.
```

Comment:
In both cases, the result is a normalized CDF function. It's just that, to properly calculate it, the input of this function has always to be "counts". So, if the input is density, we pre-multiply it by "binwidth" so that it's counts now (in the auxiliary variable then passed to the function, original input is not changed).